mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -576,15 +576,17 @@ CLASS net/minecraft/class_1297 net/minecraft/entity/Entity
 	METHOD method_5723 applyDamageEffects (Lnet/minecraft/class_1309;Lnet/minecraft/class_1297;)V
 		COMMENT Applies damage effects to {@code attacker} or {@code target}.
 		COMMENT
-		COMMENT <p>Called when {@code attacker} damages {@code target}. The attacker is
-		COMMENT usually this entity, but may be a projectile's owner entity.
+		COMMENT <p>Called when {@code attacker} damages {@code target}.
 		COMMENT
 		COMMENT <p>Used to apply damage effects based on enchantments, such
 		COMMENT as Thorns attacker damage or slowness from Bane of Arthropods.
 		COMMENT
 		COMMENT @implNote Although this method is non-static, {@code this} keyword is
 		COMMENT not used anywhere in this method.
 		ARG 1 attacker
+			COMMENT the attacker; usually this entity, but may be a {@link
```

Comment:
```suggestion
			COMMENT the attacker; usually this entity, but may be a {@linkplain
```
otherwise "projectile's owner entity" is rendered as `projectile's owner entity`

Reply:
Fixed in https://github.com/FabricMC/yarn/pull/2437/commits/e6d23ab8c102f481f588cf44487ea6431feebdd8.